.github/workflows/sync-deb-prod.yml

```diff
@@ -0,0 +1,55 @@
+name: Sync Deb artifacts to Prod
+on: repository_dispatch
```

Comment:
it's better to use an event type here to avoid execution triggered for other workflows

Reply:
This workflow would only trigger if you pass the required payload "Sync_Deb_Prod" in your dispatch event. Unless it is passed this will not run, also no other workflow uses "Sync_Deb_Prod" payload.

Reply:
The thing is the workflow starts irrespective of the payload type if it is not mentioned at the start, like in this case it will run until the conditional statement. So better to stop the trigger at first if is not for this workflow.